src/codec.ts

```diff
@@ -30,6 +30,23 @@ export interface Field {
   value: Value;
 }
 
+export interface IProtoMessageParams {
+  value: object;
+  // Function representing the message with helper methods for deserializing
+  // and serializing messages, use from generated file.
+  messageFunction?: Function;
+  // fully qualified name of the proto representing the enum definition
+  fullName: string;
```

Comment:
Just a good practice add attributes that must be passed (fullName) before those which are optional(messageFunction)